### PR TITLE
Provide easier setup

### PR DESCRIPTION
**Pull request by jeffsee55**

### CODE REVIEW

File: examples/tina-cloud-starter-experimental/pages/_app.js [hunk 1]
```diff
@@ -12,77 +12,37 @@ limitations under the License.
...
+            branch: 'main',
+            clientId: process.env.NEXT_PUBLIC_TINA_CLIENT_ID,
+            organization: process.env.NEXT_PUBLIC_ORGANIZATION_NAME,
+            isLocalClient: Boolean(
+              Number(process.env.NEXT_PUBLIC_USE_LOCAL_CLIENT)
+            ),
+          }}
+          {...pageProps}
+        >
+          {(livePageProps) => <Component {...livePageProps} />}
```
Comment (logan-anderson) [inline examples/tina-cloud-starter-experimental/pages/_app.js:40]: Would we have access to the CMS in here with `useCMS` ?

Comment (jeffsee55) [reply examples/tina-cloud-starter-experimental/pages/_app.js:40]: Yeah, I'm not sure where is best, right now you can do this:
```tsx
<TinaCMSProvider
  useUnstable={true}
  cms={(cms) => cms.doSomething}
  // and formify
  formify={(args) =>doThing}
  config={{
    branch: 'main',
    clientId: process.env.NEXT_PUBLIC_TINA_CLIENT_ID,
    organization: process.env.NEXT_PUBLIC_ORGANIZATION_NAME,
    isLocalClient: Boolean(
        Number(process.env.NEXT_PUBLIC_USE_LOCAL_CLIENT)
      ),
    }
  }
  {...pageProps}
>
```

Comment (logan-anderson) [reply examples/tina-cloud-starter-experimental/pages/_app.js:40]: I think we are already using the keyword `cms` for passing a CMS object.

**Comment by jeffsee55** [comment]
Will revisit this when things are more stable

**Pull request closed without merging**